app/controllers/document_contacts_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class DocumentContactsController < ApplicationController
+  def search
+    @document = Document.find_by_param(params[:id])
+    @options = ContactsService.new.all_by_organisation.flat_map do |org|
+      org["contacts"].map do |contact|
+        ["#{contact['title']} - #{org['name']}", contact["content_id"]]
```

Comment:
This feels a little bit weird in here as it's specific to the view code that wants to use it. Could we leave it to the view code, like we do elsewhere?